proto/cosmos/feegrant/v1beta1/tx.proto

```diff
@@ -20,6 +20,9 @@ service Msg {
   // RevokeAllowance revokes any fee allowance of granter's account that
   // has been granted to the grantee.
   rpc RevokeAllowance(MsgRevokeAllowance) returns (MsgRevokeAllowanceResponse);
+
+  // PruneAllowances prunes expired fee allowances.
+  rpc PruneAllowances(MsgPruneAllowances) returns (MsgPruneAllowancesResponse);
```

Comment:
Can we update the doc: https://github.com/cosmos/cosmos-sdk/blob/e4c0788/x/feegrant/README.md#msg-server